homeassistant/components/ipp/strings.json

```diff
@@ -26,7 +26,8 @@
     "abort": {
       "already_configured": "This printer is already configured.",
       "connection_error": "Failed to connect to printer.",
-      "connection_upgrade": "Failed to connect to printer due to connection upgrade being required."
+      "connection_upgrade": "Failed to connect to printer due to connection upgrade being required.",
+      "parse_error": "Failed to parse response from printer."
```

Comment:
@frenck string update in beta means we need to update translations and commit it to the rc branch.

Reply:
oh man now i feel bad for the extra work, sorry

Reply:
no worries, it's just running a script. but we don't always do it. Thinking now, we should actually always do it, because new translations might have come in